examples/air-quality-sensor-app/air-quality-sensor-common/include/relative-humidity-sensor-manager.h

```diff
@@ -25,7 +25,6 @@ class RelativeHumiditySensorManager
         EmberAfStatus status = RelativeHumidityMeasurement::Attributes::MeasuredValue::Set(mEndpointId, newValue);
         VerifyOrReturn(EMBER_ZCL_STATUS_SUCCESS == status,
                        ChipLogError(NotSpecified, "Failed to set RelativeHumidityMeasurement MeasuredValue attribute"));
-        ChipLogDetail(NotSpecified, "The new RelativeHumidityMeasurement value: %d", newValue);
```

Comment:
Why are these detail logs being removed, here and in other files?  This seems totally unrelated to the new example.

Reply:
You are correct, it is not related to the new example. I got carried away with the changes. I was adding error logs and at the same time noticed 3 levels of printing the same info (JSON parser, here and in the cluster layer).